charts/render.py

```diff
@@ -114,25 +120,45 @@ def keep_doc(doc):
 
 
 def main():
-    (name, namespace, values, path) = sys.argv[1:]
+    parser = argparse.ArgumentParser()
+    parser.add_argument('name', help="Denotes the name of the chart")
+    parser.add_argument('namespace', help="Metalk8s namespace to deploy unto")
```

Comment:
Let's make this optional:

```suggestion
    parser.add_argument(
        '-n',
        '--namespace',
        default="default",
        help="Namespace to deploy this chart in",
    )
```